crates/nu-protocol/src/shell_error.rs

```diff
@@ -863,7 +872,7 @@ mod tests {
                 vec!["foo", "bar", "baz"],
                 vec![
                     ("fox", Some("foo"), ""),
-                    ("FOO", Some("foo"), ""),
+                    ("FOO", None, ""),
```

Comment:
Will this lead to correct behavior if the user has mistyped the case of a command (or maybe an env var)?

Reply:
Thanks, that's a case that I never thought about..